README.rst

```diff
@@ -12,8 +12,8 @@
    :target: https://pypi.org/project/ansys-grantami-recordlists
    :alt: PyPI
 
-.. |codecov| image:: https://codecov.io/gh/pyansys/grantami-recordlists/branch/main/graph/badge.svg
-   :target: https://codecov.io/gh/pyansys/grantami-recordlists
+.. |codecov| image:: https://codecov.io/gh/ansys/grantami-recordlists/branch/main/graph/badge.svg
```

Comment:
Good spot, took me a while to figure out what the issue was here. I guess another post release?